components/rootmail.yaml

```diff
@@ -69,6 +72,131 @@ Resources:
           ]
         }
 
+  EmailGeneratorFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      InlineCode: !Sub |
+        import boto3
+        import itertools
+        import json
+        import time
+        import uuid
+
+        domain = "${Subdomain}.${Domain}"
+        ses = boto3.client("ses")
+
+        def backoff(msg, res, n):
+
+          wait = pow(2, n)
+
+          log({
+            'level': 'info',
+            'msg': msg,
+            'res': res,
+            'round': n,
+            'waiting_in_seconds': wait,
+          })
+
+          time.sleep(n)
+
+        def handler(event, context):
+
+          log({
+            'event': event,
+            'level': 'debug',
+          })
+
+          verify = event.get('verify', 'true')
```

Comment:
Not sure if mixing generator and email setup verification into one function is good? I'd rather have two functions and separation of concerns.